laravel/app/Models/Post.php

```diff
@@ -20,4 +24,19 @@ public function replies()
     {
            return $this->hasMany('App\Models\Reply');
     }
+
+    public function favorite()
+    {
+        return $this->hasMany('App\Models\Reply');
```

Comment:
参照するmodelが間違ってしまっていそうです
```suggestion
        return $this->hasMany('App\Models\Favorite');
```